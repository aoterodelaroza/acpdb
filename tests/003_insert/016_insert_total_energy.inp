## check: 016_insert_total_energy.out -a1e-10
## delete: 016_insert_total_energy.db
## labels: regression quick

verbose
system rm -f 016_insert_total_energy.db
connect 016_insert_total_energy.db

insert litref bibtex ../zz_source/bsip2.bib
insert method b3lyp/6-31+G*
 litrefs bsip1 boys1970
 description This is a method.
end
insert method test
end
insert set s22
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method b3lyp/6-31+G*
end

insert property strange
 property_type energy
 set s22
 order 1
 structures s22.c6h6_ch4
end
insert evaluation
 method b3lyp/6-31+G*
 property strange
 value -12.2
end
insert term
 method b3lyp/6-31+G*
 property strange
 atom 1
 l P
 exponent 0.1
 value -12.2
 maxcoef 10.0
end
insert calc
 property_type energy
 file ../zz_source/s22.list
 method test
end
print evaluation
print term
print full
