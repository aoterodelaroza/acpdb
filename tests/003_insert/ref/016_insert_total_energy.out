* SYSTEM: rm -f 016_insert_total_energy.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 016_insert_total_energy.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)
# INSERT LITREF bsip1
# INSERT LITREF boys1970
# INSERT LITREF s30l
# INSERT LITREF s12lb
# INSERT LITREF s12la
# INSERT LITREF risthaus2013
# INSERT LITREF xdmlarge
# INSERT LITREF johnson2009
# INSERT LITREF burns2017
# INSERT LITREF parker2015
# INSERT LITREF smith2016
# INSERT LITREF thanthiriwatte2010
# INSERT LITREF marshall2011
# INSERT LITREF burns2011
# INSERT LITREF bauza2013
# INSERT LITREF johnson2003
# INSERT LITREF wilke2009
# INSERT LITREF zhao2005
# INSERT LITREF zhao2005b
# INSERT LITREF grimme2007
# INSERT LITREF parthiban2001
# INSERT LITREF karton2011
# INSERT LITREF kozuch2013
# INSERT LITREF binkley1980
# INSERT LITREF gordon1982
# INSERT LITREF pietro1982
# INSERT LITREF dobbs1987a
# INSERT LITREF dobbs1987b
# INSERT LITREF dithcfield1971
# INSERT LITREF hehre1972
# INSERT LITREF hariharan1973
# INSERT LITREF hariharan1974
# INSERT LITREF gordon1980
# INSERT LITREF francl1982
# INSERT LITREF weigend2005
# INSERT LITREF weigend2006
# INSERT LITREF sto3g
# INSERT LITREF mini
# INSERT LITREF jensen1
# INSERT LITREF jensen2
# INSERT LITREF jensen3
# INSERT LITREF jensen4
# INSERT LITREF b3
# INSERT LITREF lyp
# INSERT LITREF dunning1
# INSERT LITREF dunning2
# INSERT LITREF grimme2010
# INSERT LITREF xdmbasis
# INSERT LITREF witte2016
# INSERT LITREF davidson1986
# INSERT LITREF andzelm1984
# INSERT LITREF kruse2012
# INSERT LITREF hbs66
# INSERT LITREF halogen
# INSERT LITREF dcacp
# INSERT LITREF qcp
# INSERT LITREF hfacp
# INSERT LITREF lasso
# INSERT LITREF osborne2000
# INSERT LITREF wb97xd1
# INSERT LITREF wb97xd2
# INSERT LITREF pepconf
# INSERT LITREF xdmhybrid
# INSERT LITREF xdm
# INSERT LITREF lcwpbe1
# INSERT LITREF lcwpbe2
# INSERT LITREF kalescky2014
# INSERT LITREF nbody1
# INSERT LITREF wateracp
# INSERT LITREF hf-3c
# INSERT LITREF kulik2016
# INSERT LITREF sure2015
# INSERT LITREF goerigk2013
# INSERT LITREF galano2006
# INSERT LITREF jensen2010
# INSERT LITREF hehre1970
# INSERT LITREF curtiss1999
# INSERT LITREF dill1975
# INSERT LITREF clark1983
# INSERT LITREF frisch1984
# INSERT LITREF rassolov1998
# INSERT LITREF mackie2011
# INSERT LITREF burns2013
# INSERT LITREF rezac2011
# INSERT LITREF dilabio2008
# INSERT LITREF mackie2008
# INSERT LITREF mackie2010
# INSERT LITREF torres2012
# INSERT LITREF dilabio2014
# INSERT LITREF vansanten2015
# INSERT LITREF goerigk2011
# INSERT LITREF aorss3
# INSERT LITREF aorss1
# INSERT LITREF aspirin
# INSERT LITREF clathrates
# INSERT LITREF aupi2
# INSERT LITREF pyricarbamate
# INSERT LITREF tmcomplex
# INSERT LITREF wallach
# INSERT LITREF junctions
# INSERT LITREF hb-piperidine
# INSERT LITREF 2nitrile
# INSERT LITREF chpi-pdxanthate
# INSERT LITREF digold
# INSERT LITREF abo3
# INSERT LITREF bipodal-carbamate
# INSERT LITREF oscil-friction
# INSERT LITREF tibond
# INSERT LITREF clathrate-hpr
# INSERT LITREF epxars-fepxas
# INSERT LITREF graphitestress-theo
# INSERT LITREF agtao3surf
# INSERT LITREF xdmchiral
# INSERT LITREF graphenemesa
# INSERT LITREF anales
# INSERT LITREF electrides
# INSERT LITREF xdmgold
# INSERT LITREF kevan
# INSERT LITREF burrito
# INSERT LITREF step
# INSERT LITREF agtao3
# INSERT LITREF xdmc9
# INSERT LITREF kaolin
# INSERT LITREF xdmbench
# INSERT LITREF ncisolids
# INSERT LITREF graphitestress
# INSERT LITREF xdmsolids
# INSERT LITREF energy-correction
# INSERT LITREF cij-partition
# INSERT LITREF supramolecular
# INSERT LITREF qhamgo
# INSERT LITREF ctc-gibbs
# INSERT LITREF gibbs2b
# INSERT LITREF gibbs2a
# INSERT LITREF sbasga2
# INSERT LITREF sbasga2-sbpga2
# INSERT LITREF aggase2
# INSERT LITREF cdal2se4
# INSERT LITREF lapla
# INSERT LITREF qtree
# INSERT LITREF variational
# INSERT LITREF sbpga2
# INSERT LITREF critic
# INSERT LITREF runwien
# INSERT LITREF b88
# INSERT LITREF s225
# INSERT LITREF s22
# INSERT LITREF s66
# INSERT LITREF s668
# INSERT LITREF g3
# INSERT LITREF s22b
# INSERT LITREF s66b
# INSERT LITREF kb49b
# INSERT LITREF kb49a
# INSERT LITREF p26
# INSERT LITREF l7
# INSERT LITREF hsg0
# INSERT LITREF ionichb
# INSERT LITREF gmtkn55
# INSERT LITREF gmtkn30
# INSERT LITREF bsr36
# INSERT LITREF pconf
# INSERT LITREF sconf
# INSERT LITREF water
# INSERT LITREF x40
# INSERT LITREF s8
# INSERT LITREF bde03
# INSERT LITREF aconf
# INSERT LITREF bhperi1
# INSERT LITREF bhperi2
# INSERT LITREF bhperi3
# INSERT LITREF bhperi4
# INSERT LITREF darc
# INSERT LITREF idisp
# INSERT LITREF s668b
# INSERT LITREF adim6
# INSERT LITREF a24
# INSERT LITREF karlsruhe1
# INSERT LITREF karlsruhe2
# INSERT LITREF pople1
# INSERT LITREF pople2
# INSERT LITREF pople3
# INSERT LITREF pople4
# INSERT LITREF pople5
# INSERT LITREF pople6
# INSERT LITREF pople7
# INSERT LITREF pople8
# INSERT LITREF bonifacic1974
# INSERT LITREF brauer2014
# INSERT LITREF clementi1982
# INSERT LITREF stevens1984
# INSERT LITREF topiol1980
# INSERT LITREF topiol1981
# INSERT LITREF duijneveldt1994
# INSERT LITREF davidson1994
# INSERT LITREF xdmpoly
# INSERT LITREF critic2
# INSERT LITREF kahn1972
# INSERT LITREF kahn1976
# INSERT LITREF haynes2006
# INSERT LITREF faver2011
# INSERT LITREF faver2012
# INSERT LITREF jensen2013
# INSERT LITREF goerigk2014
# INSERT LITREF cao2011
# INSERT LITREF dolg2011
# INSERT LITREF kulik2016b
# INSERT LITREF mentel2013
# INSERT LITREF mao2016
# INSERT LITREF deng2011
# INSERT LITREF gutowski1993
# INSERT LITREF xdmsurface
# INSERT LITREF carter2014
# INSERT LITREF linear1
# INSERT LITREF linear2
# INSERT LITREF linear3
# INSERT LITREF spackman2001
# INSERT LITREF linear4
# INSERT LITREF whitten1973
# INSERT LITREF dunlap1979
# INSERT LITREF feyereisen1993
# INSERT LITREF vahtras1993
# INSERT LITREF weigend2002
# INSERT LITREF sodt2006
# INSERT LITREF gpu1
# INSERT LITREF gpu2
# INSERT LITREF gpu3
# INSERT LITREF liechtenstein1995
# INSERT LITREF dudarev1998
# INSERT LITREF kabsch1976
# INSERT LITREF kaminsky2007
# INSERT LITREF maier2015
# INSERT LITREF hornak2006
# INSERT LITREF bsip2

* INSERT: insert data into the database (METHOD)
# INSERT METHOD b3lyp/6-31+G*

* INSERT: insert data into the database (METHOD)
# INSERT METHOD test

* INSERT: insert data into the database (SET)
# INSERT SET s22
# INSERT STRUCTURE s22.nh3_nh3
# INSERT STRUCTURE s22.nh3_nh3_1
# INSERT STRUCTURE s22.nh3_nh3_2
# INSERT PROPERTY s22.nh3_nh3
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.nh3_nh3)
# INSERT STRUCTURE s22.h2o_h2o
# INSERT STRUCTURE s22.h2o_h2o_1
# INSERT STRUCTURE s22.h2o_h2o_2
# INSERT PROPERTY s22.h2o_h2o
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.h2o_h2o)
# INSERT STRUCTURE s22.h2co2_h2co2
# INSERT STRUCTURE s22.h2co2_h2co2_1
# INSERT STRUCTURE s22.h2co2_h2co2_2
# INSERT PROPERTY s22.h2co2_h2co2
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.h2co2_h2co2)
# INSERT STRUCTURE s22.formamide_formamide
# INSERT STRUCTURE s22.formamide_formamide_1
# INSERT STRUCTURE s22.formamide_formamide_2
# INSERT PROPERTY s22.formamide_formamide
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.formamide_formamide)
# INSERT STRUCTURE s22.uracil_uracil_hb
# INSERT STRUCTURE s22.uracil_uracil_hb_1
# INSERT STRUCTURE s22.uracil_uracil_hb_2
# INSERT PROPERTY s22.uracil_uracil_hb
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.uracil_uracil_hb)
# INSERT STRUCTURE s22.pyridoxine_aminopyridine
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_1
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_2
# INSERT PROPERTY s22.pyridoxine_aminopyridine
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.pyridoxine_aminopyridine)
# INSERT STRUCTURE s22.adenine_thymine_wcc1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_2
# INSERT PROPERTY s22.adenine_thymine_wcc1
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.adenine_thymine_wcc1)
# INSERT STRUCTURE s22.ch4_ch4
# INSERT STRUCTURE s22.ch4_ch4_1
# INSERT STRUCTURE s22.ch4_ch4_2
# INSERT PROPERTY s22.ch4_ch4
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.ch4_ch4)
# INSERT STRUCTURE s22.c2h4_c2h4
# INSERT STRUCTURE s22.c2h4_c2h4_1
# INSERT STRUCTURE s22.c2h4_c2h4_2
# INSERT PROPERTY s22.c2h4_c2h4
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c2h4_c2h4)
# INSERT STRUCTURE s22.c6h6_ch4
# INSERT STRUCTURE s22.c6h6_ch4_1
# INSERT STRUCTURE s22.c6h6_ch4_2
# INSERT PROPERTY s22.c6h6_ch4
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_ch4)
# INSERT STRUCTURE s22.c6h6_c6h6_pd
# INSERT STRUCTURE s22.c6h6_c6h6_pd_1
# INSERT STRUCTURE s22.c6h6_c6h6_pd_2
# INSERT PROPERTY s22.c6h6_c6h6_pd
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_c6h6_pd)
# INSERT STRUCTURE s22.pyrazine_pyrazine
# INSERT STRUCTURE s22.pyrazine_pyrazine_1
# INSERT STRUCTURE s22.pyrazine_pyrazine_2
# INSERT PROPERTY s22.pyrazine_pyrazine
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.pyrazine_pyrazine)
# INSERT STRUCTURE s22.uracil_uracil_stack
# INSERT STRUCTURE s22.uracil_uracil_stack_1
# INSERT STRUCTURE s22.uracil_uracil_stack_2
# INSERT PROPERTY s22.uracil_uracil_stack
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.uracil_uracil_stack)
# INSERT STRUCTURE s22.indole_c6h6_stack
# INSERT STRUCTURE s22.indole_c6h6_stack_1
# INSERT STRUCTURE s22.indole_c6h6_stack_2
# INSERT PROPERTY s22.indole_c6h6_stack
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.indole_c6h6_stack)
# INSERT STRUCTURE s22.adenine_thymine_stack
# INSERT STRUCTURE s22.adenine_thymine_stack_1
# INSERT STRUCTURE s22.adenine_thymine_stack_2
# INSERT PROPERTY s22.adenine_thymine_stack
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.adenine_thymine_stack)
# INSERT STRUCTURE s22.c2h4_c2h2
# INSERT STRUCTURE s22.c2h4_c2h2_1
# INSERT STRUCTURE s22.c2h4_c2h2_2
# INSERT PROPERTY s22.c2h4_c2h2
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c2h4_c2h2)
# INSERT STRUCTURE s22.c6h6_h2o
# INSERT STRUCTURE s22.c6h6_h2o_1
# INSERT STRUCTURE s22.c6h6_h2o_2
# INSERT PROPERTY s22.c6h6_h2o
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_h2o)
# INSERT STRUCTURE s22.c6h6_nh3
# INSERT STRUCTURE s22.c6h6_nh3_1
# INSERT STRUCTURE s22.c6h6_nh3_2
# INSERT PROPERTY s22.c6h6_nh3
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_nh3)
# INSERT STRUCTURE s22.c6h6_hcn
# INSERT STRUCTURE s22.c6h6_hcn_1
# INSERT STRUCTURE s22.c6h6_hcn_2
# INSERT PROPERTY s22.c6h6_hcn
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_hcn)
# INSERT STRUCTURE s22.c6h6_c6h6_t
# INSERT STRUCTURE s22.c6h6_c6h6_t_1
# INSERT STRUCTURE s22.c6h6_c6h6_t_2
# INSERT PROPERTY s22.c6h6_c6h6_t
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.c6h6_c6h6_t)
# INSERT STRUCTURE s22.indole_c6h6_t
# INSERT STRUCTURE s22.indole_c6h6_t_1
# INSERT STRUCTURE s22.indole_c6h6_t_2
# INSERT PROPERTY s22.indole_c6h6_t
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.indole_c6h6_t)
# INSERT STRUCTURE s22.phenol_phenol
# INSERT STRUCTURE s22.phenol_phenol_1
# INSERT STRUCTURE s22.phenol_phenol_2
# INSERT PROPERTY s22.phenol_phenol
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=s22.phenol_phenol)

* INSERT: insert data into the database (PROPERTY)
# INSERT PROPERTY strange

* INSERT: insert data into the database (EVALUATION)
# INSERT EVALUATION (method=b3lyp/6-31+G*;property=strange)

* INSERT: insert data into the database (TERM)
# INSERT TERM (method=b3lyp/6-31+G*;property=strange;atom=1;l=P;exponent=0.1)

* INSERT: insert data into the database (CALC)
# INSERT EVALUATION (method=test;property=23;nvalue=1)

* PRINT: print the contents of the database 

| methodid| propid| #values| 1st value|
| 1| 1| 1| -3.133|
| 1| 2| 1| -4.989|
| 1| 3| 1| -18.753|
| 1| 4| 1| -16.062|
| 1| 5| 1| -20.641|
| 1| 6| 1| -16.934|
| 1| 7| 1| -16.66|
| 1| 8| 1| -0.527|
| 1| 9| 1| -1.472|
| 1| 10| 1| -1.448|
| 1| 11| 1| -2.654|
| 1| 12| 1| -4.255|
| 1| 13| 1| -9.805|
| 1| 14| 1| -4.524|
| 1| 15| 1| -11.73|
| 1| 16| 1| -1.496|
| 1| 17| 1| -3.275|
| 1| 18| 1| -2.312|
| 1| 19| 1| -4.541|
| 1| 20| 1| -2.717|
| 1| 21| 1| -5.627|
| 1| 22| 1| -7.097|
| 1| 23| 1| -12.2|
| 2| 23| 1| 28|

* PRINT: print the contents of the database 

| methodid| propid| atom| l| exponent| #values| 1st value| maxcoef|
| 1| 23| 1| 1| 0.1| 1| -12.2| 10|

* PRINT: print the contents of the database 

# Table of property types
| id | key | description |
| 1 | ENERGY_DIFFERENCE | A difference of molecular or crystal energies (reaction energy, binding energy, lattice energy, etc.) |
| 2 | ENERGY | The total energy of a molecule or crystal |
| 3 | DIPOLE | The electric dipole of a molecule |
| 4 | STRESS | The stress tensor in a crystal |
| 5 | D1E | The first derivatives of the energy wrt the atomic positions in a molecule or crystal |
| 6 | D2E | The second derivatives of the energy wrt the atomic positions in a molecule or crystal |
| 7 | HOMO | The orbital energy of the highest occupied molecular orbital |
| 8 | LUMO | The orbital energy of the lowest unoccupied molecular orbital |

# Number of literature references: 234

# Table of methods
| id| key| gaussian_keyword| psi4_keyword| litrefs| description|
| 1| b3lyp/6-31+G*| method=b3lyp/6-31+G*;gbs=| method=b3lyp;basis=6-31+G*| bsip1 boys1970| This is a method.|
| 2| test| | | | |

# Table of sets
| id| key| litrefs| description|
| 1| s22| bsip1 boys1970 | A description.|

# Number of properties and structures in each set
| id | key | properties | structures |
| 1 | s22 | 23 | 66 |

# Evaluations and terms for each combination of set & method
| set-id | set-key | method-id | method-key | #evaluations | #terms |
| 1 | s22 | 1 | b3lyp/6-31+G* |23 | 1 |
| 1 | s22 | 2 | test |1 | 0 |

## Description of the training set
# The TRAINING SET is NOT DEFINED
--- No atoms found (ATOM) ---
--- No angular momenta found (LMAX) ---
--- No exponents found (EXP) ---
--- No subsets found (SUBSET) ---
--- No weights found (W) ---
--- No empty method found (EMPTY) ---
--- No reference method found (REFERENCE) ---

## Table of saved training sets in the database
| Name |

